lib/krane/kubernetes_resource/deployment.rb

```diff
@@ -91,6 +91,10 @@ def deploy_timed_out?
       return false if deploy_failed?
       return super if timeout_override
 
+      if progress_condition.present?
+        StatsD.client.increment('kubectl.error', 1, tags: statsd_tags)
+      end
```

Comment:
I'm confused, why are we incrementing `kubectl.error`?

Reply:
I wanted to use an existing metric in Krane. Do you have any suggestions as to which metric we can use? We can also create a new metric for this

Reply:
I don't think there's a metric we have that's quite suitable. Perhaps I'm ignorant, but is there some reason against using a bespoke metric?

Reply:
No reason against it! I've updated the metric to a new one 👍